docs/user/guides/setup/index.rst

```diff
@@ -16,9 +16,8 @@ The following how-to guides help you solve common tasks and challenges in the se
 ⏩️ :doc:`Using custom URL redirects in documentation projects </guides/redirects>`
     Configuring your Read the Docs project for redirecting visitors from one location to another.
 
-⏩️ :doc:`Managing subprojects </guides/subprojects>`
-    Need several projects under the same umbrella?
-    Start using subprojects, which is a way to host multiple projects under a "main project".
+⏩️ :doc:`/guides/subprojects`
```

Comment:
We should probably just redo all of these. Every time I see the rewritten header 2x it makes me die a bit inside.